### PR TITLE
Removed PK in JS SDK and tested all code samples

### PR DESCRIPTION
**Pull request by krofax**
Removed private key values in JS SDK and tested all code samples.

### CODE REVIEW

File: docs/api/js/accounts-l1-l2.md [hunk 1]
```diff
@@ -209,7 +209,8 @@ async requestExecute(transaction: {
 import * as zksync from "zksync-web3";
 import { BigNumber, ethers } from "ethers";
 
-const PRIVATE_KEY = "0xc8acb475bb76a4b8ee36ea4d0e516a755a17fad2e84427d5559b37b544d9ba5a";
+const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
+const CONTRACT_ADDRESS = " "
```
Comment (kailash-manasarovar) [inline docs/api/js/accounts-l1-l2.md:213]: add placeholder

File: docs/api/js/accounts.md [hunk 2]
```diff
@@ -185,19 +188,25 @@ async getBalanceL1(token?: Address, blockTag?: ethers.providers.BlockTag): Promi
 import * as zksync from "zksync-web3";
 import { ethers } from "ethers";
 
-const PRIVATE_KEY = "0xc8acb475bb76a4b8ee36ea4d0e516a755a17fad2e84427d5559b37b544d9ba5a";
+const PRIVATE_KEY = "0x1f09ccee64f523d43139fd56a034b0c9bde80624b8a1bbbe69019ebad4c96195";
```
Comment (kailash-manasarovar) [inline docs/api/js/accounts.md:191]: ?

File: docs/api/js/providers.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ import { Provider } from "zksync-web3";
 
 const provider = new Provider("https://testnet.era.zksync.dev");
 
-const USDC_L2_ADDRESS = "0x0faF6df7054946141266420b43783387A78d82A9";
+const USDC_L2_ADDRESS = "<YOUR_USDC_L2_AADRESS>";
```
Comment (kailash-manasarovar) [inline docs/api/js/providers.md:72]: spelling

**Comment by krofax** [comment]
@kailash-manasarovar please approve

**Comment by kailash-manasarovar** [comment]
> @kailash-manasarovar please approve

Morning @krofax you changed a fair bit of code here. Please get a tech review first.

File: docs/api/js/accounts-l1-l2.md [hunk 4]
```diff
@@ -39,13 +39,13 @@ async approveERC20(
 import * as zksync from "zksync-web3";
 import { ethers } from "ethers";
 
-const PRIVATE_KEY = "0xc8acb475bb76a4b8ee36ea4d0e516a755a17fad2e84427d5559b37b544d9ba5a";
+const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
 
 const zkSyncProvider = new zksync.Provider("https://testnet.era.zksync.dev/");
 const ethereumProvider = ethers.getDefaultProvider("goerli");
 const wallet = new zksync.Wallet(PRIVATE_KEY, zkSyncProvider, ethereumProvider);
```
Comment (pcheremu) [inline docs/api/js/accounts-l1-l2.md:46]: Somewhere I see that we keep 
```
import * as zksync from "zksync-web3";
new zksync.Provider(); or new zksync.Wallet();
```
structure.

Somewhere I see updated:
```
import { Wallet, Provider } from "zksync-web3";
new Provider(); or new Wallet();
```

I would suggest to use one standard in all examples.

File: docs/api/js/accounts-l1-l2.md [hunk 5]
```diff
@@ -39,13 +39,13 @@ async approveERC20(
 import * as zksync from "zksync-web3";
 import { ethers } from "ethers";
 
-const PRIVATE_KEY = "0xc8acb475bb76a4b8ee36ea4d0e516a755a17fad2e84427d5559b37b544d9ba5a";
+const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
```
[the remark this answers is not in the document]
Comment (krofax) [reply docs/api/js/accounts-l1-l2.md:42]: Obtaining a private key is beyond the scope of this sample code. It's assumed to be basic knowledge and I don't think it should be here.

[the remark this answers is not in the document]
Comment (pcheremu) [reply docs/api/js/accounts-l1-l2.md:42]: Oh, I have confused you by pointing to the wrong code line. I mentioned about USDC_ADDRESS f.e. on the line 48. Perhaps it also may be beyond the scope.

[the remark this answers is not in the document]
Comment (pcheremu) [reply docs/api/js/accounts-l1-l2.md:42]: btw, is this address belong to the general testnet address of the corresponding testnet or this is a user own address?

File: docs/api/js/providers.md [hunk 6]
```diff
@@ -69,7 +69,7 @@ import { Provider } from "zksync-web3";
 
 const provider = new Provider("https://testnet.era.zksync.dev");
 
-const USDC_L2_ADDRESS = "0x0faF6df7054946141266420b43783387A78d82A9";
+const USDC_L2_ADDRESS = "<YOUR_USDC_L2_ADDRESS>";
 // Getting  USDC balance of account 0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5 at the latest processed block
 console.log(await provider.getBalance("0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5", "latest", USDC_L2_ADDRESS));
```
Comment (pcheremu) [inline docs/api/js/providers.md:74]: Please double check, I see here we have an issue as everywhere now... https://zksync2-testnet.zkscan.io/tokens 0x0faF6df7054946141266420b43783387A78d82A9 - is a common token, if we are going to remove it everywhere, we should mention the link where to find this address: https://zksync2-testnet.zkscan.io/tokens. 
image
but 0x0614BB23D91625E60c24AAD6a2E6e2c03461ebC5 is a private address, we can delete it and comment that user should use its own here.

File: docs/api/js/accounts-l1-l2.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (krofax) [reply docs/api/js/accounts-l1-l2.md:42]: The user's own address.

File: docs/api/js/providers.md (hunk 6 above)
Comment (krofax) [reply docs/api/js/providers.md]: @pcheremu I've updated it.

**Review by pcheremu — APPROVED** [review]

**Pull request merged**